el/solid.go

```diff
@@ -0,0 +1,13 @@
+package el
```

Comment:
Could you just add these to `content.go`? That's where they're grouped at https://developer.mozilla.org/en-US/docs/Web/HTML/Element .

Reply:
Sure!

Reply:
I moved them and also got rid of the dependency on `attr`. I think when you test `el` it should still pass if something is wrong with the `attr` package (yeah I am nitpicking here).

Reply:
Sounds reasonable.